spec/01-abstract.md

```diff
@@ -1,7 +1,39 @@
-Distributed tracing is a set of tools and practices to monitor the health and reliability of a distributed application. A distributed application is an application that consists of multiple components that are deployed and operated separately. It is also known as micro-service.
+# Problem Statement

```

Comment:
I'd suggest to close this PR https://github.com/w3c/trace-context/pull/221 to avoid double-reviewing

Reply:
Done